test/functional/find.test.js

```diff
@@ -1655,7 +1655,7 @@ describe('Find', function () {
     }
   });
 
-  it('shouldCorrectlyIterateOverCollection', {
+  it.skip('shouldCorrectlyIterateOverCollection', {
```

Comment:
Needs unskip ticket

Reply:
Added todo comment to https://jira.mongodb.org/browse/NODE-3819